src/braket/aws/aws_quantum_task_batch.py

```diff
@@ -0,0 +1,231 @@
+# Copyright 2019-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+from __future__ import annotations
+
+import time
+from concurrent.futures.thread import ThreadPoolExecutor
+from typing import List, Set, Union
+
+from braket.annealing import Problem
+from braket.aws.aws_quantum_task import AwsQuantumTask
+from braket.aws.aws_session import AwsSession
+from braket.circuits import Circuit
+
+
+class AwsQuantumTaskBatch:
+    """Executes and a batch of quantum tasks in parallel.
```

Comment:
Remove "and", unless you were planning to add something else.

Reply:
Dangit